File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.java
import java.util.*;
class Solution {
    public List<Integer> solution(int l, int r) {
        List<Integer> ret = new ArrayList<>();
        for(int i = l; i <= r; i++){
            boolean flag = false;
            String s = i + "";
                for(char c : s.toCharArray()){
                    if(c == '0' || c == '5') continue;
                    flag = true;
                    break;
                }
                if(flag) continue;
                ret.add(i);
            }
        if(ret.size() == 0) ret.add(-1);
        return ret;
    }
}

File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    ret = []
    for i in range(l, r + 1):
        flag = False
        for c in str(i):
            if c != '0' and c != '5':
                flag = True
            if flag:
                break
        if not flag:
            ret.append(i)
    return [-1] if not ret else ret